patches/chromium/revert_use_more_binaries_from_clang_toolchain_in_mac_build.patch

```diff
@@ -0,0 +1,97 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Keeley Hammond <khammond@slack-corp.com>
+Date: Wed, 16 Jul 2025 20:19:56 -0700
+Subject: Revert "Use more binaries from clang toolchain in mac build"
+
+This reverts commit 17b850a9f8e913fb5e465d895f545b069def57d1.
```

Comment:
Can you add context to why this was reverted

Reply:
Oh yes, sorry! Chrome moved to bundling their otool and nm binaries into the third-party/llvm-build folder, which I could not get to regen correctly in our own CI. We do explicitly delete and regenerate this folder in `fix-sync/action.yml#L25`, so I may have missed a specific regen step we need, but I reverted it after a few attempts to keep us moving with Chromium

Will add this to the patch description and include an upgrades follow up issue.